views/about.ejs

```diff
@@ -1 +1,121 @@
-<h1> <%= title %> Home Page</h1>
+<link rel="stylesheet" href="css/about.css">
+<div class="container text-justify">
+<div class="row justify-content-center">
+    <div class="col-12 col-md-10 col-sm-12 ">
+        <h3>About the LADC Society</h3>
+        <br>
+        <p>The Signet Society is the place at Harvard where students and faculty who are interested in the arts can meet over lunch and 
+            without academic constraints. Special events such as readings, talks, screenings, shows, and receptions reflect the interests of 
+            undergraduates, graduate, and associate members alike: literature, the performing arts, the fine arts, the social sciences, politics, 
+            and culture. Renowned artists, writers, actors, and scholars visiting Cambridge often join Signet members for discussions or to give
+             more formal presentations. The Signet Society Alumni work to connect Signet alumni living in 35 countries with the Harvard community 
+             and each other.</p>
+    <br>
+    <br>
```

Comment:
Try using margin instead of multiple <br> tags

Reply:
ok sir, i will make the changes.

Reply:
Done sir